app/model/category/Add.scala

```diff
@@ -0,0 +1,14 @@
+package model.category
+
+import play.api.data.Form
+import play.api.data.Forms._
+
+import model.common.ViewValueCommon
+
+case class ViewValueCategoryAdd (
+  title:    String                = "カテゴリーの追加",
+  cssSrc:   Seq[String]           = Seq("main.css"),
+  jsSrc:    Seq[String]           = Seq("main.js"),
+  form:     Form[CategoryForm],
+  allColor: Seq[(String, String)] = ViewValueCategory.allColor
+) extends ViewValueCommon
```

Comment:
修正は不要ですが
こう書くこともできます

```suggestion
case class ViewValueCategoryAdd (
  form:     Form[CategoryForm],
) extends ViewValueCommon {

  val title:    String                = "カテゴリーの追加"
  val cssSrc:   Seq[String]           = Seq("main.css")
  val jsSrc:    Seq[String]           = Seq("main.js")
  val allColor: Seq[(String, String)] = ViewValueCategory.allColor
}
```


case classの中はスッキリしそうですね